.changeset/config.json

```diff
@@ -0,0 +1,30 @@
+{
+  "$schema": "https://unpkg.com/@changesets/config@2.0.0/schema.json",
+  "changelog": ["@changesets/cli/changelog", { "repo": "remix-run/remix" }],
+  "commit": false,
+  "fixed": [
+    [
```

Comment:
Every package will remain version-fixed for now *except* for `create-remix`, which is never installed as a dependency and should generally be releasable independently.